core/data/src/main/java/com/iteneum/OfficeInfoItemModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.iteneum
+
+import com.itenum.domain.ContactTelephone
+import com.itenum.domain.Email
+
+data class OfficeInfoItemModel(
```

Comment:
Why you create this model and contactTelephone model and EmailModel? Create only one model. 

We must move the logic of your model to domain of your module, in core only logic is added that can be reused

Reply:
the application models the case of having multiple emails of contact, if there are departements associated to the bussines then the class email easily can handle that.

in this use case, the purpose is to show or present info into a screen,  that is the main reason and purpose of this class , that is why it is needed